services/database/serverless.yml

```diff
@@ -15,6 +15,7 @@ custom:
       - master
       - val
       - production
+  dynamodbBackupEnabled: ${ssm:/configuration/${self:custom.stage}/dynamodbBackup/enabled~true, ssm:/configuration/default/dynamodbBackup/enabled~true}
```

Comment:
I'd like this to be as is, but with one additional entry...
```
dynamodbBackupEnabled: ${ssm:/configuration/${self:custom.stage}/dynamodbBackup/enabled~true, ssm:/configuration/default/dynamodbBackup/enabled~true, true}
```
This way, the default setting with no SSM parameters set is true/enabled.
This way, there is no requirement to set this SSM parameter (so change the readme above too).
But, it is fully configurable via SSM if necessary

Reply:
Also, do the checklist check boxes if you would, which should guide you to assigning this PR, adding reviewers, and adding it to the proejct board.

Reply:
@joeaero19 Taking a step back, I'm not sure why we wouldn't enable PITR for all environments.  I'm not sure a configuration override is even called for here.

I vote hardcode it:
```
PointInTimeRecoveryEnabled: true
```

Reply:
@mdial89f I set PointInTimeRecoveryEnabled to true by hardcoding it.